nmdc_runtime/site/changesheets/base.py

```diff
@@ -0,0 +1,80 @@
+# nmdc_runtime/site/changesheets/base.py
+"""
+base.py: Provides data classes for creating changesheets for NMDC database objects.
+"""
+
+import logging
+import time
+from dataclasses import dataclass
+from pathlib import Path
+import requests
+from typing import Any, ClassVar, Dict
+
+from nmdc_runtime.site.resources import RuntimeApiUserClient
+
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s %(levelname)s %(" "message)s"
+)
+
+JSON_OBJECT = Dict[str, Any]
```

Comment:
I like this type alias. I find it easier to mentally process than the `Dict[str, Any]` that it aliases.

I assumed the variable defined on the line below it was also a type alias (due to its proximity and similar formatting), but I don't think it is. I suggest adding `: TypeAlias` to the first one; i.e.

```diff
- from typing import Any, ClassVar, Dict
+ from typing import Any, ClassVar, Dict, TypeAlias

(...)

- JSON_OBJECT = Dict[str, Any]
+ JSON_OBJECT: TypeAlias = Dict[str, Any]
```
> Docs: https://docs.python.org/3/library/typing.html#typing.TypeAlias